include/Page/Base.php

```diff
@@ -229,7 +229,8 @@ public function modify_admin_footer_text( $footer_text ) {
 	 * Enqueue JavaScript.
 	 */
 	public function enqueue_script() {
-		wp_enqueue_script( 'bulk-move', $this->get_plugin_dir_url() . 'assets/js/src/bulk-move.js', array( 'jquery', 'postbox' ), BM_BulkMove::VERSION, true );
+		wp_register_script( 'select2', $this->get_plugin_dir_url() . 'assets/js/select2.min.js' );
```

Comment:
Add jquery as dependency to `select2` library and also include its version. You can find the version that we are using from the `packages.json` file.